itou/users/models.py

```diff
@@ -358,6 +358,24 @@ def get_full_name(self):
         full_name = f"{self.first_name.strip().title()} {self.last_name.upper()}"
         return full_name.strip()[:70]
 
+    def get_redacted_full_name(self):
+        """
+        Return full name in redacted form for privacy, e.g. J** H***h
+        """
+
+        def get_letters_for_name(name):
+            name = name.strip()[:10]
```

Comment:
Il faudrait mieux ne pas tronquer à 10, et plutot limiter le nombre de `*`

-> Charlemagne devrait devenir `C*********e` et pas `C*********n`
-> Jean-Charles devient `J********s`  et pas `J********l`


Il faudrait en revanche probablement itérer sur les prénoms ou noms : 
Johann Sebastian Bach devrait devenir `J****n S*******n B**h` et non `J********n B**h`